testing/functional/tests/cypress/page-objects/student-objects/ApplicationObject.ts

```diff
@@ -0,0 +1,279 @@
+    cy.get("//tbody/tr/td[5]").each(($el, index, $list) => {
+      if ($el.text() === "Draft") {
+        $el.click();
+      }
+    });
+  }
+
+  applicationButton() {
+    return cy.xpath("//button[normalize-space()='Applications']");
+  }
+
+  verifyApplicationText() {
+    return cy.contains("My Applications");
+  }
+
+  startNewApplicationButton() {
+    return cy.contains("Start New Application");
+  }
+
+  //<StartRegion----------------------------Start New Application------------------------------>
+  verifyStartNewApplicationText() {
+    return cy.contains("Select an application to begin");
+  }
+
+  startApplicationStudyYearDisableButton() {
+    return cy.xpath("//button[@disabled='disabled']");
+  }
+
+  selectStudyYearDropdown() {
+    cy.fixture("newApplicationForm").then((testData) => {
+      cy.get(".form-group").eq(1).click({ force: true });
+      cy.wait(1000);
+      cy.get(".choices__input.choices__input--cloned")
+        .click({ force: true })
+        .type(testData.selectStudyYear)
+        .type("{enter}");
+    });
+    cy.wait(2000);
+  }
+
+  startApplicationStudyYearEnableButton() {
+    return cy.xpath("//button[@disabled='disabled']");
+  }
+
+  removedButton() {
+    return cy.xpath("//button[normalize-space()='Remove item']");
+  }
+
+  //Alert message in "Already In Progress"
+  applicationAlreadyInProgressText() {
+    return cy.contains("Application already in progress");
+  }
+
+  applicationAlreadyInProgressTextCard() {
+    return cy.get(".v-card-actions > .v-btn");
+  }
+
+  //Start Application
+  startApplicationTuitionWaiverButton() {
+    return cy.xpath("//button[@name='data[startApplication1]']");
+  }
+  startApplicationBCloanForgivenessProgramButton() {
+    return cy.xpath("//button[@name='data[startApplication2]']").click();
+  }
+  startApplicationPacificLeaderLoanForgivenessButton() {
+    return cy.xpath("//button[@name='data[startApplication4]']").click();
+  }
+  startApplicationInterestFreeStatusButton() {
+    return cy.xpath("//button[@name='data[startApplication3]']").click();
+  }
+
+  //<EndRegion----------------------------Start New Application------------------------------>
+
+  //<StartRegion--------------------------Draft Status------------------------------------->
+
+  draftApplication() {
+    return cy.get(":nth-child(1) > .svg-inline--fa > path");
+  }
+
+  draftApplicationVerifyText() {
+    return cy.contains("Let’s get started on your application");
+  }
+
+  NextSectionButton() {
+    return cy.get(":nth-child(3) > .ml-auto > .v-btn");
+  }
+
+  PreviousSectionButton() {
+    return cy.contains("Previous section");
+  }
+
+  errorMsgTxtForSchoolAttending() {
+    return cy.contains("The school I will be attending is required");
+  }
+
+  selectionDropdownInvalid() {
+    cy.get(".form-control.ui.fluid.selection.dropdown.is-invalid");
+  }
+
+  WelcomeText() {
+    return cy.contains("Let’s get started on your application");
+  }
+
+  WelcomeButton() {
```

Comment:
Please check the methods cases to be camel cases.

Reply:
Changes have been done.